app/Repositories/Interfaces/PayloadRepositoryInterface.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace App\Repositories\Interfaces;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.OpenBraceNewLine</br>**Severity**: error</br>**File**: app/Repositories/Interfaces/PayloadRepositoryInterface.php</br>**Line**: 5</br>Opening brace of a interface must be on the line after the definition